examples/02-registered-shopper-auth-token.ts

```diff
@@ -8,74 +8,54 @@
 /**
  * Get authorization token for a registered customer
  * Usage: ts-node examples/02-registered-shopper-auth-token.ts
- * Note: Replace configuration parameters and user credentials before running
+ * Note: Replace configuration parameters and client credentials before running
  */
 import { ClientConfig, Customer } from "../dist";
-import {
-  getObjectFromResponse,
-  ResponseError,
-  ShopperToken,
-  stripBearer,
-} from "@commerce-apps/core";
-import ShopperCustomers = Customer.ShopperCustomers;
+import { getObjectFromResponse } from "@commerce-apps/core";
 
-//client configuration parameters
+// client credentials
+const CLIENT_ID = "da422690-7800-41d1-8ee4-3ce983961078";
+const CLIENT_SECRET = "D*HHUrgO2%qADp2JTIUi";
+
+// client configuration parameters
 const clientConfig: ClientConfig = {
   parameters: {
-    clientId: "your-client-id",
-    organizationId: "your-org-id",
-    shortCode: "your-short-code",
-    siteId: "your-site-id",
+    clientId: CLIENT_ID,
+    organizationId: "f_ecom_zzte_053",
+    shortCode: "kv7kzm78",
+    siteId: "RefArch",
   },
 };
 
-//customer credentials
-const username = "username";
-const password = "password";
-
 /**
  * Get token for the registered customer
  *
  * @returns authorization token
  */
-async function getRegisteredShopperToken(): Promise<
-  ShopperToken<ShopperCustomers.Customer>
-> {
-  const credentials = `${username}:${password}`;
-  const buff = Buffer.from(credentials);
-  const base64data = buff.toString("base64");
+async function getRegisteredShopperToken(): Promise<string> {
+  const credentials = `${CLIENT_ID}:${CLIENT_SECRET}`;
+  const base64data = Buffer.from(credentials).toString("base64");
   const headers = { Authorization: `Basic ${base64data}` };
-
-  const client = new ShopperCustomers(clientConfig);
-
-  const response = await client.authorizeCustomer(
-    { headers: headers, body: { type: "credentials" } },
+  const client = new Customer.ShopperLogin(clientConfig);
+
+  const response = await client.getAccessToken(
+    {
+      headers,
+      body: {
+        /* eslint-disable-next-line @typescript-eslint/camelcase */
```

Comment:
```suggestion
```
We should avoid including things in the example that are specific to this project. You can add an override to the eslintrc so that we can suppress the error outside of this file.